### PR TITLE
Ryt tmp

### PR DESCRIPTION
**Pull request by declmal**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

File: cvm/quantization/tfm_ops.py [hunk 1]
```diff
@@ -179,8 +195,59 @@ def compile(self, op, **kwargs):
...
+    "ssd0_convpredictor6_conv0_fwd",
+    "mrt_rewrite_ssd0_expand_bn2_0",
+    "ssd0_convpredictor9_conv0_fwd",
+    "mrt_rewrite_ssd0_expand_trans_bn3_0",
+    "ssd0_convpredictor8_conv0_fwd",
+    "mrt_rewrite_ssd0_expand_bn3_0",
+    "ssd0_convpredictor11_conv0_fwd",
+    "ssd0_convpredictor10_conv0_fwd",
+}
+
```
Comment (wlt-cortex) [inline cvm/quantization/tfm_ops.py]: May the restore names be useless and should be commented on or deleted?

File: cvm/quantization/transformer.py [hunk 2]
```diff
@@ -314,25 +323,47 @@ def get_output_scales(self, base_oscales, maps):
...
+    _sym, _prm = model.symbol, model.params
+    _sym, _prm = tpass.attach_input_shape(
+        _sym, _prm, input_shapes)
+
+    print("Before fixxing shape: ",
+          calculate_ops(_sym, _prm, normalize=False))
+    _sym, _prm = prepare_for_compile(_sym, _prm)
+    _sym, _prm = fuse_constant(_sym, _prm)
+    print("After fixxing shape: ",
+          calculate_ops(_sym, _prm, normalize=False))
```
Comment (wlt-cortex) [inline cvm/quantization/transformer.py]: Remove the debug code.

File: cvm/quantization/tfm_ops.py (hunk 1 above)
Comment (declmal) [reply cvm/quantization/tfm_ops.py]: Already deleted.

File: cvm/quantization/transformer.py (hunk 2 above)
Comment (declmal) [reply cvm/quantization/transformer.py]: Already removed.

**Pull request merged**